### ISSUE TITLE
Chore: Distribution files should just be dist

### ISSUE THREAD

**Issue by simonireilly** [LyraSearch/lyra#97]
Looking at the entry points of the distribution:

https://github.com/nearform/lyra/blob/6d4fd373b40e834fc0cfd7fe633460ac0a85b4e0/packages/lyra/package.json#L22-L24

The exports will be dist folder.

The actual package includes multiple CI / Testing fixtures / coverage report files that are not needed and likely bloat the package install.

Image shows all files included that are not necessary:

![image](https://user-images.githubusercontent.com/30017294/184643226-3132349a-7c62-4673-bcce-4f1090836a89.png)

## Suggested change

Utilise the files declaration to denote which files should actually be included in the distribution.

- Source: https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files

Required change:

```json5
  // packages/lyra/package.json
  "files": [
    "dist"
  ]
```

**Comment by micheleriva** [comment]
This is related to https://github.com/nearform/lyra/pull/87. We'll try to fix this in the next release

**Comment by simonireilly** [comment]
Oh dear, I should have checked, thanks

**Comment by micheleriva** [comment]
Fixed in v0.1.3
